src/codegen.rs

```diff
@@ -1043,6 +1051,12 @@ add_with_broadcasting((float*) {out}, (float*) {X}, (float*) {Y}, (int*)  {out_s
 
                     Some(add_out)
                 }
+                RelayOperator::RelayLeakyReLU => todo!(),
```

Comment:
Can you add `#[ignore]`d tests in this module for these new operators? (See comment below about `#[ignore]`.) You can just copy and paste the tests for the other operators and modify them!